domain/src/EventSubscriber/DomainSubscriber.php

```diff
@@ -74,22 +74,20 @@ public function __construct(DomainNegotiatorInterface $negotiator, DomainLoaderI
    *
    * @param \Symfony\Component\HttpKernel\Event\GetResponseEvent $event
    *   The Event to process.
+   *
+   * @see domain_alias_domain_request_alter
    */
   public function onKernelRequestDomain(GetResponseEvent $event) {
-    $redirect = FALSE;
     // Negotiate the request and set domain context.
     /** @var \Drupal\domain\DomainInterface $domain */
     if ($domain = $this->domainNegotiator->getActiveDomain(TRUE)) {
       $domain_url = $domain->getUrl();
       if ($domain_url) {
-        $redirect_type = $domain->getRedirect();
```

Comment:
The reasoning on name here is because the redirect could be different, though I think status is perfectly clear.

The only code that sets a redirect for a domain is in domain_alias_domain_request_alter().